doc: prepare new release 4.1.2 (#5413)
tracim/tracim: changelog.md

> # 4.1.2 / 2022-02-14

Reviewer: make sure you update the date here @raphj

Author: ```suggestion
# 4.1.2 / 2022-02-15
```

This maintenance release fixes a few things:

>  - the Docker image, broken after a setuptools update, once again works

Reviewer 2: ```suggestion
 - the experimental Docker image for arm64, broken after a setuptools update, once again works
```

>  - on mobile devices, a work around prevents the send button to be covered by the browser's native UI

Reviewer 3: ```suggestion
 - on mobile devices, a workaround prevents the send button to be covered by the browser's native UI
```
